contrib/Docker_Grafana_Blueflood/docker-entrypoint.sh

```diff
@@ -37,6 +37,8 @@ blueflood:
   apikey: $RAX_APIKEY                     
   authentication_module: rax_auth
   authentication_class: BluefloodAuth
+  enable_submetrics: true
+  submetric_aliases: { "_sum": 'sum' }
```

Comment:
@wolfdancer I am just concerned that if someone spins up blueflood and starts playing with it, they would have difficulty with graphs as we dont have any documentation about submetrics and this is not regular behavior of grafana.